src/Screen/X11/TopWindow.cpp

```diff
@@ -115,10 +115,24 @@ TopWindow::CreateNative(const TCHAR *text, PixelSize size,
     const Atom atoms[] = {
       XInternAtom(x_display, "_NET_WM_STATE_FULLSCREEN", false),
     };
+#if defined(SELFBUILD_RASPBIAN) || defined(SELFBUILD_ARMBIAN)
+    XEvent e;
+    memset(&e, 0, sizeof(e));
+    e.xclient.type = ClientMessage;
+    e.xclient.message_type = XInternAtom(x_display, "_NET_WM_STATE", false);
+    e.xclient.display = x_display;
+    e.xclient.window = x_window;
+    e.xclient.format = 32;
+    e.xclient.data.l[0] = 1 /* _NET_WM_STATE_ADD */;
+    e.xclient.data.l[1] = (long)atoms[0];
+    XSendEvent(x_display, DefaultRootWindow(x_display), False,
+	       SubstructureRedirectMask, &e);
```

Comment:
What's this?

Reply:
This is for fullscreen, i.e. do same as XInternAtom(x_display, "_NET_WM_STATE_FULLSCREEN", false). However, XInternAtom() does not give me fullscreen on Raspbian and Armbian.

Reply:
But nothing of this is specific to Raspbian and it's unrelated to the rest of this commit *and* it's not documented in your commit message. You may propose such a change, but not hide it in an unrelated PR.

Reply:
Move this code to another pull request.